src/chains/base.ts

```diff
@@ -70,6 +71,7 @@ export enum ChainKey {
   FTMT = 'ftmt',
   LNAT = 'lnat',
   SCLT = 'sclt',
+  MODT = 'modt',
```

Comment:
there is a ticket to remove the testnets from our types and BE
https://lifi.atlassian.net/browse/LF-7854
I guess we shouldn't add the new testnet here, wdyt?🤔

Reply:
noted, forgot this ticket, will remove